SubmersedVR/Input/SteamVrGameInput.cs

```diff
@@ -27,7 +27,8 @@ public static bool ShouldIgnore(GameInput.Button button)
                 || button == GameInput.Button.Slot3
                 || button == GameInput.Button.Slot4
                 || button == GameInput.Button.Slot5
-                || button == GameInput.Button.AutoMove;
+                || button == GameInput.Button.AutoMove
+                || button.ToString() == "45" || button.ToString() == "46";
```

Comment:
What are those buttons? Could you please add a comment?

Reply:
Ive been trying to get the newest Nitrox working with Submersed Vr and Nitrox added mappings for those two things which was causing constant errors to be thrown. So I was just bypassing those errors with this code.